Documentation/Books/Manual/Programs/Arangod/Log.md

```diff
@@ -89,6 +89,15 @@ To additionally log HTTP request to a file named "requests.log" add the options:
 --log.level requests=info --log.output requests=file:///path/to/requests.log
 ```
 
+If you specify `--log.file-mode octalvalue` then any newly created log
+file will use "octalvalue" as file mode. Please note that the `umask`
+value will be applied as well.
+
+If you specify `--log.file-group name` then any newly created log file
+will try to use "name" as group name. Please note that you have to be
+a member of that group. Otherwise the group ownership will not be
+changed.
+
```

Comment:
maybe add a hint here that the option is not available on some platforms, e.g. Windows...?